src/main/webapp/i18n/en/participation.json

```diff
@@ -2,18 +2,18 @@
     "artemisApp": {
         "participation": {
             "home": {
-                "title": "Participants",
-                "createLabel": "Create a new participant",
-                "createOrEditLabel": "Create or edit a participant"
+                "title": "Participation",
+                "createLabel": "Create a student's participation",
+                "createOrEditLabel": "Create or edit a student's participation"
             },
-            "created": "Created new participant with identifier {{ param }}",
-            "updated": "Updated participant {{ param }}",
-            "deleted": "Deleted participant {{ param }}",
+            "created": "Created new participation with identifier {{ param }}",
+            "updated": "Updated {{ param }}'s participation",
+            "deleted": "Deleted {{ param }}'s participation",
```

Comment:
I am not sure, if with identifier {{ param }} would be better for updated and deleted as well... But if I am correct, these translations are currently not used anyways.
All other changes that were made with 2dd4bce0c85fbd93b3abf1d782d679d05b89e150 seem good to me 👍